css/properties/column-fill.json

```diff
@@ -57,6 +57,70 @@
             "deprecated": false
           }
         },
+        "auto": {
+          "__compat": {
+            "support": {
+              "chrome": {
+                "version_added": "50"
+              },
+              "chrome_android": "mirror",
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "≤72"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "oculus": "mirror",
+              "opera": "mirror",
+              "opera_android": "mirror",
+              "safari": {
+                "version_added": "8"
```

Comment:
Here you decided for 8 instead of 9. This is inconsistent with the decision you made in https://github.com/mdn/browser-compat-data/pull/22305#discussion_r1512495055 for the descendant data problem.